1208-projects/project_wizard.feature

```diff
@@ -1,29 +1,16 @@
 Feature: User edits Project
-    Background: A User is logged in with relevant access
 
     In order to manage a projects content
     I want to be able to manage all relevant information
 
     In order to achive a low mental load on the user
     I want to achive a high degree of recognition between the desing of Projects registration wizard and Publications registration wizard
 
-    Rule: Any User can create a project, becoming origin Project Owner
-
-    Rule: The Project Owner can grant the Project Manager role 
-
-    Rule: The Project Owner, the Project Manager and there Curators are the custodian of the project
-
-    Rule: A project can only have one Project Owner, Project Manager and Coordinating Institution - at any given time
-
-    Rule: The Project Manager reside from the Coordinating Institution, but exceptions exists
-
-    Rule: A project have Participents, like the Local Project Manager
-
-    Rule: The Local Project Manager is a Project Manager at his institutions' scope of the project
-
-    Rule: A project may have several different sources of funding
+    Background: A User is logged in with relevant access
+    # The User has a role defined in the project 
+    # or is a Curator for one of the defined users
 
-    Rule: Funding organisazations names a funding a project, this is a funding source to our project
+    Rule: Any User can create a project, becoming origin Project Owner
```

Comment:
hvis vi skal bruke `Rule` bør vi ha innrykk, for å enklere se hva som ligger i hver regel: https://cucumber.io/docs/gherkin/reference/#rule

```
Feature: Dette er en Feature
    
    Background:
        Given (...)

    Rule: Dette er en Regel

        Sceanrio: Dette er et Scenario
            Given (...)

        Sceanrio: Dette er et annet Scenario
            Given (...)
```

Reply:
Done